### PR TITLE
DX-1507 Add webhooks manager as an example app

### PR DESCRIPTION
**Pull request by janza**

### CODE REVIEW

**Comment by anthonyroux** [comment]
@bishopwm, you might be interested in this.

File: examples/webhooks-manager/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+          Subscribe to a webhook using this UI. Your endpoint must be an HTTPS
+          URL with a valid SSL certificate that can correctly process event
+          notifications.{" "}
+        </p>
+        <p>
+          When adding a webhook you will receive an HTTP POST request with a
+          challenge in the body on the callback URL that you provide.
+        </p>
+        <p>
+          You must respond to this request from your backend and ensure that you
```
Comment (marcospinello) [inline examples/webhooks-manager/pages/index.tsx:201]:
```suggestion
          Your backend must respond to this request. The response your backend sends
```

File: examples/webhooks-manager/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+          send the same challenge in the response.
```
Comment (marcospinello) [inline examples/webhooks-manager/pages/index.tsx:202]:
```suggestion
          must contain the same challenge.
```

**Review by marcospinello — CHANGES_REQUESTED** [review]
Left some comments, they're all suggestions: feel free to accept or reject them!

**Pull request merged**